### ISSUE TITLE
what is this error?

### ISSUE THREAD

**Issue by LucasGSGS** [spyder-ide/spyder#6661]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/Users/shuogong/anaconda3/lib/python3.6/site-packages/spyder/plugins/editor.py", line 1724, in new
    text, enc = encoding.read(self.TEMPLATE_PATH)
  File "/Users/shuogong/anaconda3/lib/python3.6/site-packages/spyder/utils/encoding.py", line 243, in read
    text, encoding = decode( open(filename, 'rb').read() )
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shuogong/.spyder-py3/template.py'

## Version and main components

* Spyder Version: 3.2.6 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Darwin

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.22.0 (OK)
numpy >=1.7      :  1.14.0 (OK)
sphinx >=0.6.6   :  1.6.6 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.11.1 (OK)
psutil >=0.3     :  5.4.3 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.27.3 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.2.1 (OK)
pylint >=0.25    :  1.8.2 (OK)

```

**Comment by ccordoba12** [comment]
This is a duplicate of issue #5717 and it'll be fixed in our next version (3.2.8)

**Comment by LucasGSGS** [comment]
Thx!
